site.json

```
@@ -30,7 +30,7 @@
     "index": {
       "startDate": "2024-06-26T00:00:00.000Z",
       "endDate": "2024-07-02T00:00:00.000Z",
```

```suggestion
      "endDate": "2024-07-04T00:00:00.000Z",
```

---

It would be better to set the time later in the day if the plan is to release on the 4th, otherwise the banner will disappear soon after midnight (e.g. the banner is not visible on the website now because the endDate was 14 hours ago).
```suggestion
      "endDate": "2024-07-04T23:59:00.000Z",
```